client/my-sites/domains/domain-management/transfer/transfer-out/index.jsx

```diff
@@ -42,15 +43,18 @@ class Transfer extends React.Component {
 		const {
 			currentUserCanManage,
 			isPendingIcannVerification,
-			transferAwayEligibleAtMoment,
+			transferAwayEligibleAtDate,
 		} = getSelectedDomain( this.props );
 		let section = null;
 
 		if ( ! currentUserCanManage ) {
 			section = NonOwnerCard;
 		} else if ( transferProhibited ) {
 			section = TransferProhibited;
-		} else if ( transferAwayEligibleAtMoment && transferAwayEligibleAtMoment.isAfter() ) {
+		} else if (
+			transferAwayEligibleAtDate &&
+			this.props.moment( transferAwayEligibleAtDate ).isAfter()
```

Comment:
Are there any other `moment` usages in this component? I don't see any. This one only compares two dates and doesn't need localization, does it? A simple `import moment` should work.

Reply:
Yup, you're right, there's only a date comparison here. Done.